stats/test/handler.test.js

```diff
@@ -434,6 +435,37 @@ describe('HTTP request handler', () => {
     })
   })
 
+  describe('GET /retrieval-result-status/daily', () => {
+    it('returns daily retrieval result status for the given date range', async () => {
+      await pgPools.stats.query(`
+        INSERT INTO daily_retrieval_result_status
+        (day, status, rate)
+        VALUES
+        ('2024-01-11', 'Error1', 0.1),
+        ('2024-01-11', 'Error2', 0.9),
+        ('2024-01-12', 'Error1', 1),
+        ('2024-01-13', 'Error1', 0.5),
+        ('2024-01-13', 'Error3', 0.5)
```

Comment:
Ditto, please use real retrieval result codes and rates.

Reply:
f75b30b